sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, metric='mean_test_score',
```

Comment:
Which one is easier code? The GridSearchCV after this method is added? I think the idea of ``validation_curve`` was to have ``validation_curve`` be easier to plot then ``GridSearchCV``. Also, it doesn't have an estimator interface and is supposed to be easier to use? Though I was never really sold on ``validation_curve``. Maybe @agramfort can explain?